ui-tests/pageobjects/page.ts

```diff
@@ -58,6 +66,7 @@ export default class Page {
 
       tableData.push(rowData);
     }
+    console.log(tableData);
```

Comment:
best not to do this if possible, don't want to accidentally log something we shouldn't